MailerQ/RestApi/Client/IRestApiClientResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Net;
+
+namespace MailerQ.RestApi.Client
+{
+    public interface IRestApiClientResponse
```

Comment:
@ms-darianbenito Could you add the summary documentation?

Reply:
Done!